### PR TITLE
PW-48: Update test env build instructions

### PR DESCRIPTION
**Pull request by Sophist-UK**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ cd picard-website
 cp website/config.py.example website/config.py
 ```
 
-Edit website/config.py so that PLUGINS_REPOSITORY points to your local copy of the picard-plugins repository:
+Edit website/config.py so that PLUGINS_BUILD_DIR points to your local temporary directory:
```
Comment (samj1912) [inline INSTALL.md]: It is not supposed to point to a local temporary directory. It's where the plugin files will be generated to.

File: website/config.py.example [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+#CHANGELOG_CACHE_TIMEOUT = 5 * 60
+#
+#SUPPORTED_LANGUAGES = [
+#    'en',
+#    'fr',
+#]
+#
 #SERVER_HOSTNAME = "127.0.0.1"
-#SERVER_PORT = 6060
+#SERVER_PORT = 6060
```
Comment (samj1912) [inline website/config.py.example]: Add an endline

**Comment by samj1912** [comment]
Please refer to https://github.com/metabrainz/picard-website/pull/59 for documentation updates

File: INSTALL.md (hunk 1 above)
Comment (Sophist-UK) [inline INSTALL.md]: How about a .git-ignore(d) subdirectory then?

File: website/config.py.example (hunk 2 above)
Comment (Sophist-UK) [inline website/config.py.example]: Ok.

File: INSTALL.md (hunk 1 above)
Comment (samj1912) [inline INSTALL.md]: It already is. Supposed to be :/build

**Pull request merged**